docs/reference/setup/install/docker.asciidoc

```diff
@@ -43,7 +43,51 @@ docker pull {docker-repo}:{version}
 
 endif::[]
 
-Now that you have the {es} Docker image, you can start a
+==== Verify the {es} Docker image signature
```

Comment:
I wonder if we should flag this as "optional". This isn't a required step, only for those that want to validate signatures.